manifests/g/Getscreenme/Dashboard/1.0.16/Getscreenme.Dashboard.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Getscreenme.Dashboard
```

```suggestion
PackageIdentifier: Getscreenme.Dashboard.Beta
```
PackageName and InstallerUrl suggests the package is currently in beta.